exercises/simple-cipher/HINT.md

```diff
@@ -0,0 +1,9 @@
+This exercise has three test classes: StepOne.java, StepTwo.java and StepThree.java. To run all test classes:
+```
+$ gradle test
+```
```

Comment:
This is rather subtle, but the "Track Hint" [SETUP.md](https://github.com/exercism/xjava/blob/master/SETUP.md) is inserted before the "Implementation Hint" (this file, `HINT.md`), so you'll want to consider the contents of the first file too.

In that merge, this instruction to run `gradle test` will appear twice.